src/main/java/io/github/christophermanahan/carnitas/HTTPResponse.java

```diff
@@ -0,0 +1,14 @@
+package io.github.christophermanahan.carnitas;
+
+public class HTTPResponse implements Response {
+
+  static final String VERSION = "HTTP/1.1";
+  static final String GET = "200 OK";
+  static final String CRLF = "\r\n";
+
+  public HTTPResponse(String data) {}
```

Comment:
I believe this is here to make it easier to fit into `Optional.map`, yeah?.  Since we don't know how this class is going to shake out yet, I am reluctant to do this quite yet.